packages/client/src/components/Tabs/index.js

```diff
@@ -66,34 +103,32 @@ export default function VTabs(props) {
 
     return (
         <div className={classes.root}>
-            <Tabs
+            <ProcessTabs
                 /* sets horizontal or vertical tabs */
                 orientation="vertical"
                 /* changes type of tab feature if tabs > height*/
                 // variant="scrollable"
+                indicatorColor="primary"
                 value={value}
-                onChange={handleChange}
+                onChange={handleSelect}
                 aria-label="Tabs Example"
                 className={classes.tabs}
             >
-                {
-                    labels.map(
-                        label => (
-                            <Tab label={label} {...a11yProps({ value })} />
-                        )
-                    )
-                }
-
-            </Tabs>
-            {
-                texts.map(
-                    (text, index) => (
-                        <TabPanel value={value} index={index}>
-                            {text}
-                        </TabPanel>
-                    )
-                )
-            }
+                {labels.map((label, index) => (
+                    <Tab
+                        key={index}
+                        label={label}
+                        style={index === value ? color : defaultColor}
```

Comment:
we actually don't need to track the `color` state, we can just do `index === value ? selected : defaultColor`